components/webpack/webpack.config.js

```diff
@@ -206,7 +206,7 @@ module.exports = async function (env, argv) {
           },
         },
         {
-          resourceQuery: /raw/,
+          test: /\.html/,
```

Comment:
@petemill FYI
If it isn't acceptable, let's refix it in another PR.
This one should be merged now to fix rebuilding issues that affects every day dev experience.